### DISCUSSION TITLE
"Sync brightness changes from Built-in and Apple displays" seems not work for me.

### DISCUSSION THREAD

**Discussion by Lotus-95** [MonitorControl/MonitorControl#1148] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Discussion

I turn on the "Sync brightness changes from Built-in and Apple displays", but the brightness seems not sync between apple  and build-in displays. But the slider adjusts brightness works.

**Comment by waydabber** [comment]
The feature meant to synchronize only system induced changes (like the ambient light sensor, the touch bar or the macOS native brightness slider). If you change the built in display brightness in MonitorControl manually, it is intentionally not synchronized. Only the change (delta) is synchronized, not the absolute brightness values/percentages.

**Reply by ninjit** [reply]
I've been wondering about this too.
At least for all my various monitors at home and work that I use regularly, setting the sliders to reflect the same/similar brightness %s seems to be perceptually matching to me.
